crypto/rsa/rsa.c

```diff
@@ -121,23 +121,17 @@ int rsa_new_end(RSA *rsa, BN_CTX *ctx) {
    * BN_FLG_CONSTTIME flag. */
```

Comment:
This comment should be replaced with an assertion that the relevant variables have the CONSTTIME flag set.. But, see my other comment below.